recordings-downloader/config.yml

```diff
@@ -24,18 +24,18 @@ steps:
     Some SDKs will include helper functions to obtain the token using the Client ID and Client Secret, while on other SDKs, the HTTP Request and Response needs to be manually handled to obtain the token.
 
     For this tutorial, Client Credentials Grant Type will be used as user context is not needed in acquiring the desired information
-    More information about OAuth and Authorization could be found in the [documentation](https://developer.mypurecloud.com/api/rest/authorization/)
+    More information about OAuth and Authorization could be found in the [documentation](https://developer.mypurecloud.com/api/rest/authorization/).
```

Comment:
Remove the domain from the link to avoid redirecting from regional domains. i.e. just `[documentation](/api/rest/authorization/)`